### PR TITLE
Show recording time

### PR DESCRIPTION
**Pull request by holiber**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -131,6 +131,7 @@
     "lodash-decorators": "4.3.1",
     "mime": "2.4.3",
     "moment": "2.22.2",
+    "moment-duration-format": "^2.3.2",
```
Comment (andycreeth) [inline package.json]: I don't think we really need to add a new library to the codebase.  The streaming service already has a function that does this called `formattedDurationSince`.  Look at how we do this for streaming timer.

Comment (holiber) [reply package.json]: It's actually just a plugin for moment.js

Will use `formattedDurationSince`

File: app/components/StudioFooter.vue.ts [hunk 2]
```diff
@@ -35,9 +36,16 @@ export default class StudioFooterComponent extends Vue {
...
+  recordingTime = '';
 
   mounted() {
     this.confirmYoutubeEnabled();
+
+    // update recording time
+    setInterval(() => {
+      if (!this.streamingService.isRecording) return;
+      this.recordingTime = this.streamingService.formattedDurationInCurrentRecordingState;
+    }, 1000);
```
Comment (andycreeth) [inline app/components/StudioFooter.vue.ts]: You should clear this interval in the `destroyed` hook.  I know the footer is unlikely to ever unmount, but it's good practice to ensure we never leak interval handlers.

**Comment by andycreeth** [comment]
Please fix so we can merge this in.

**Pull request merged**